pnpm-lock.yaml

```diff
@@ -1,6105 +1,1412 @@
-lockfileVersion: '9.0'
+lockfileVersion: '6.0'
```

Comment:
Woops 😬

Reply:
if only there were some way to declare which `packageManager` a project uses :thinking: 

haha

Reply:
https://github.com/ember-cli/ember-cli/pull/10584